src/components/__tests__/Subheader.test.js

```diff
@@ -6,8 +6,8 @@ import "@testing-library/jest-dom/extend-expect";
 import renderer from "react-test-renderer";
 
 afterEach(cleanup);
-const dim = ["Datasource", "Campaign"];
-const met = ["Clicks", "Impressions"];
+const dim = ["datasource", "campaign"];
+const met = ["clicks", "impressions"];
```

Comment:
no abbreviations please for both dimensions and metrics

Reply:
07a7f7ac05e79bb2e92b595ee7c906f69e8c2473